packages/feedback-spa/src/app/app.module.ts

```diff
@@ -18,6 +18,8 @@ import {FeedbackItemComponent} from './feedback/feedback-item/feedback-item.comp
 import { CountBoxComponent } from './feedback/count-box/count-box.component';
 import { DropdownComponent } from './feedback/dropdown/dropdown.component';
 import { GraphQLModule } from './graphql.module';
+import '@one-platform/opc-footer/dist/opc-footer';
+import '@one-platform/opc-header/dist/opc-header';
```

Comment:
Nothing wrong here but what's been bothering me about opc=-components is the need to import the component from the `/dist` directory for individual components? Why is that so? Why aren't we simply packaging the `dist/` directory in the release components?

Reply:
These would be added in the new components